ccp/vars

```diff
@@ -7,6 +7,12 @@ SUPPORT_EMAIL=support-ccp@dkfz-heidelberg.de
 PRIVATEKEYFILENAME=/etc/bridgehead/pki/${SITE_ID}.priv.pem
 
 BROKER_URL_FOR_PREREQ=$BROKER_URL
+DEFAULT_LANGUAGE=DE
+DEFAULT_LANGUAGE_LOWER_CASE=${DEFAULT_LANGUAGE,,}
+ENABLE_EXPORTER=true
+ENABLE_LOGIN=true
+ENABLE_TEILER=true
+#ENABLE_DATASHIELD=true
```

Comment:
Should the EXPORTER, TEILER be all activated automatically? I think at least datashield should be added only in the sites configuration like mtba, because it will need an installation due to jq and the permission problems with /tmp/bridgehead and /var/cache/bridgehead

Reply:
The teiler and the exporter should be in always in CCP. DataSHIELD also as soon as we enter in production.